testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/AuthServerTestEnricher.java

```diff
@@ -586,51 +586,54 @@ public static void setJsseSecurityProviderForOutboundSslConnectionsOfElytronClie
             "Determining the JSSE security provider to use for outbound " +
             "SSL/TLS connections of the Elytron client..."
         );
-        /** First locate the wildfly-config.xml to use. Per:
-         *  https://docs.wildfly.org/21/Client_Guide.html#wildfly-config-xml-discovery
-         *
-         *  1) try to load it from the 'wildfly.config.url' property
-         */
-        String wildflyConfigXmlPath = System.getProperty("wildfly.config.url");
 
-        //  2) If not set, scan the classpath
+        // Use path to wildfly-config.xml directly if specified
+        String wildflyConfigXmlPath =
+            System.getProperty("path.to.wildfly.config");
+
+        // Otherwise scan the classpath to determine its location
         if (wildflyConfigXmlPath == null) {
             log.debug("Scanning classpath to locate wildfly-config.xml...");
             final String javaClassPath = System.getProperty("java.class.path");
-            for (String dir : javaClassPath.split(":")) {
+            for (String dir : javaClassPath.split(File.pathSeparator)) {
                 if (!dir.isEmpty()) {
                     String candidatePath = dir + File.separator +
                         "wildfly-config.xml";
                     if (new File(candidatePath).exists()) {
                         wildflyConfigXmlPath = candidatePath;
                         log.debugf(
-                            "Using wildfly-config.xml at '%s' location!",
+                            "Found wildfly-config.xml at the '%s' location!",
```

Comment:
No exclamation marks in logs please
```suggestion
                            "Found wildfly-config.xml at '%s' location",
```

Reply:
Removed them (there at the location pointed out, and also at couple of other places in the same routine).